nixos/tests/fish.nix

```diff
@@ -10,6 +10,8 @@ import ./make-test-python.nix ({ pkgs, ... }: {
         coreutils
         procps # kill collides with coreutils' to test https://github.com/NixOS/nixpkgs/issues/56432
       ];
+      # TODO: remove if/when #267880 is merged and this is a default
+      services.logrotate.enable = false;
```

Comment:
This is a bit out of scope of this PR don't you think?

Maybe it deserve its own commit.

Reply:
> This is a bit out of scope of this PR don't you think?

definitely. i just couldn't run this test otherwise due to the issues mentioned there and thought it may be nice to add for others

> Maybe it deserve its own commit.

would that be acceptable in this PR as well or should i make a second? i'll split it for now in any case